package.json

```diff
@@ -69,8 +68,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-intl": "^5.7.0",
-    "react-redux": "*",
     "react-router-dom": "^5.2.0",
-    "redux": "*"
+    "redux": "^4.0.0"
```

Comment:
While at it `redux` can be dropped too, right? I dont see it getting used anywhere.